<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RNFtpManager</class>
 <widget class="QDialog" name="RNFtpManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Data Path:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="datapathEdit"/>
               </item>
               <item>
                <widget class="QPushButton" name="brwserBtn">
                 <property name="maximumSize">
                  <size>
                   <width>36</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Max Work Together Nodes:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QCheckBox" name="timePeriodCbx">
                 <property name="text">
                  <string>Download By Time Period</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="timePeriodFrame">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                  <string>SelectAll/Clear</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableView" name="tableView"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>9</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>晶振主频TC：</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="TClineEdit">
                  <property name="text">
                   <string>3145728000</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Sampe Frequncy:</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QComboBox" name="comboBox">
                   <item>
                    <property name="text">
                     <string>1000</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>500</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>250</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="downloadBtn">
                <property name="text">
                 <string>&amp;DownLoad
Files</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clearMemBtn">
                <property name="text">
                 <string>&amp;Clear 
Memory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="setSampleFreqBtn">
                <property name="text">
                 <string>&amp;Set
Sample Frequency</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopCurrentBtn">
                <property name="text">
                 <string>&amp;Stop 
Current Work</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="refreshBtn">
             <property name="text">
              <string>&amp;Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="closeBtn">
             <property name="text">
              <string>&amp;Close</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="clearBtn">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
